packages/aws-cdk/README.md

```diff
@@ -372,6 +372,44 @@ For this reason, only use it for development purposes.
 **⚠ Note #2**: This command is considered experimental,
 and might have breaking changes in the future.
 
+### `cdk watch`
+
+The `watch` command is similar to `deploy`,
```

Comment:
```suggestion
The `watch` command is similar to `deploy --hotswap`,
```

Reply:
It's explained below that `watch` implies `--hotswap`. Also, you can use `watch` without `--hotswap`, which is also explained below.

If you'd still like to to include `--hotswap` here, I won't oppose it, but I feel like it's a little misleading.

Reply:
Okay fair enough